.gitignore

```diff
@@ -13,3 +13,6 @@ htmlcov
 
 # OS detritus
 *.DS_Store
+
+dist/
+sftp-config.json
```

Comment:
This isn't a file that has anything to do with codejail.  You can exclude in a global git ignore if it's something you often end up with in your working trees.